### PR TITLE
[refactor] Split up Event component

### PR DESCRIPTION
**Pull request by MikeVedsted**
**NOTE**
this relies on the latest proposed changes to findAllEvents service.

I've split up the event component to be easier to read and maintain. 
The buttons look weird because the size is 100% **and** it has margins on top of that. I've commented it out, until we have decided on how to go forward with the button styling. 

Hopefully this will make it a lot easier to make changes to the event component in the future.

### CODE REVIEW

**Review by Chiranjibichapagain — APPROVED** [review]
Great work Mike. Thanks :)

File: client/src/components/Event/Event.scss [hunk 1]
```diff
@@ -101,36 +35,13 @@
...
-  }
-}
-
 @media only screen and (max-width: 1024px) {
   .event {
-    &__actions {
-      margin-left: 300px;
+    &__buttons {
+      display: inline;
+      & * {
```
Comment (Rostislav013) [inline client/src/components/Event/Event.scss:42]: Does it need space between here?

File: client/src/util/helperFunctions.tsx [hunk 2]
```diff
@@ -2,16 +2,16 @@ export const calculateEventAge = (eventDate: string) => {
   const diff = new Date().getTime() - new Date(eventDate).getTime()
   const hours = parseInt((diff / 3600000).toFixed(0))
   if (hours <= 24) {
-    return hours + 'h'
+    return hours + ' hours'
```
Comment (Rostislav013) [inline client/src/util/helperFunctions.tsx:5]: Can add 'hour' for 1 hour

File: client/src/util/helperFunctions.tsx [hunk 3]
```diff
@@ -2,16 +2,16 @@ export const calculateEventAge = (eventDate: string) => {
...
-    return days + 'd'
+    return days + ' days'
   } else if (hours > 730 && hours <= 8640) {
     const months = (hours / 730).toFixed()
-    return months + 'm'
+    return months + ' months'
   } else {
     const years = (hours / 8640).toFixed(0)
-    return years + 'y'
+    return years + ' years'
```
Comment (Rostislav013) [inline client/src/util/helperFunctions.tsx:14]: Can add single day, month, year.

btw are we gonna keep a year old event? Shall we set a limit to keep events n then delete?

**Review by Rostislav013 — APPROVED** [review]

**Pull request merged**